README.md

```diff
@@ -201,6 +201,13 @@ var railsImagePath = "<%= helpers.image_path('rails.png') %>";
 
 This is enabled by the `rails-erb-loader` loader rule in `config/webpack/shared.js`.
 
+## Ready for Elm
```

Comment:
[Fixed](https://github.com/bkuhlmann/webpacker/commit/8a9e9e177d2b7e22c4c6324cc80324dc1a630ca6) (rebased). I've moved it to the bottom just before *Troubleshooting* section.